A3-Antistasi/Templates/A3-WotPTemplate.Tanoa/description.ext

```diff
@@ -1,11 +1,8 @@
 #include "defines.hpp"
 #include "dialogs.hpp"
-#include "GOM_defines.hpp"
-#include "GOM_functions.hpp"
-#include "GOM_dialogs.hpp"
 
-author="Barbolani";
-OnLoadName = "Tropical Antistasi";
+author="Barbolani Cleaned and fixed by duke";
```

Comment:
Author should be a single username. Would be worth doing a PR for a credit roll instead.